apps/test/solutions/applab/ec_screens.js

```diff
@@ -158,7 +152,8 @@ module.exports = {
 
         ReactTestUtils.Simulate.click($("#design-properties button").eq(-1)[0]);
```

Comment:
Oh, I see this is the "Yes" button. Never mind, the comments are ok as they were I think